docs/hybrid_applications.md

```diff
@@ -0,0 +1,71 @@
+# Hybrid applications
+
+## Overview
+
+You can use Datadog on the same application both from the React Native and the native sides.
+
+RUM Events will be reported as coming from the same application and the same source in Datadog RUM.
+
+## Constraints
+
+A few constraints to keep in mind:
+
+1. You cannot initialize more than 1 instance of the native `DatadogSDK` as it uses a singleton pattern on the native side. That means that once you initialize the native SDK on the native side or on the RN side (by calling `DdSdkReactNative.initialize`) it is configured for both.
+2. If you try to report Datadog RUM events or logs before the initialization on iOS, future RUM events and logs won't be sent.
```

Comment:
Perhaps some context on why this happens would be beneficial for clients to know.

Reply:
It's purely for architectural reasons that are going to disappear with SDK v2. Should I go into more details about this?

Reply:
If that's the case then it's fine leaving as is.